src/util/config.ts

```diff
@@ -74,6 +77,19 @@ const envConfig = {
   MAX_RETRY_ATTEMPTS: process.env.MAX_RETRY_ATTEMPTS
     ? parseInt(process.env.MAX_RETRY_ATTEMPTS, 10)
     : 3,
+  PAGOPA_MAIL_LOGO_URI: process.env.PAGOPA_MAIL_LOGO_URI
+    ? process.env.PAGOPA_MAIL_LOGO_URI
+    : "",
+  PERSONAL_DATA_VAULT_API_BASE_PATH: process.env
+    .PERSONAL_DATA_VAULT_API_BASE_PATH
+    ? process.env.PERSONAL_DATA_VAULT_API_BASE_PATH
+    : "",
+  PERSONAL_DATA_VAULT_API_HOST: process.env.PERSONAL_DATA_VAULT_API_HOST
+    ? process.env.PERSONAL_DATA_VAULT_API_HOST
+    : "",
+  PERSONAL_DATA_VAULT_API_KEY: process.env.PERSONAL_DATA_VAULT_API_KEY
+    ? process.env.PERSONAL_DATA_VAULT_API_KEY
+    : "",
```

Comment:
Does this make `getResultOrThrow` throw an exception if no value is provided?
Otherwise, I would suggest replacing this with:

```suggestion
  PERSONAL_DATA_VAULT_API_KEY: process.env.PERSONAL_DATA_VAULT_API_KEY
    ? process.env.PERSONAL_DATA_VAULT_API_KEY
    : throw new Error("No Personal Data Vault API key provided!"),
```

Because there is no sensible fallback value for this if it's not provided (same for PDV host)

Reply:
Fix with e1721baa863ef659248512a13a3e7929d8425d7e !
if we delete the default values automatically in case of missing parameters an exception is launched as required by your comment 😎

Reply:
Wait, am I wrong or this way even when the env vars are set they are not set into the config object?

Reply:
No since the property list used of the Config object is as follows.
 https://github.com/pagopa/pagopa-notifications-service/blob/ca7bd35b0eee3b2a9d3774f63c782559bf1af57e/src/util/config.ts#L37

envConfig is used to value property values by default. 
Having removed the PDV variables from this list and not specifying them in the env file we will have an exception when starting the service.

Reply:
Fixed in 7074adebb08c9bf2ebda1f56291ffdfc2b894969 I assume 🙂